src/docs/self-hosted/backup.mdx

```diff
@@ -81,6 +81,49 @@ scopes:
 For more detailed information, including how to encrypt a backup, see `sentry-admin.sh export
 --help`.
 
+#### Encryption
+
+The `sentry-admin.sh` CLI supports encryption and decryption of backups using 2048-bit asymmetric
+RSA keys. These keys can be stored manually as files accessible from the local file system:
+
+```shell
+# Encrypt and backup using locally stored keys.
+sentry-admin.sh export global /path/to/export.tar --encrypt-with /path/to/public/key/rsa.pub
```

Comment:
Should this be `./sentry-admin.sh`? Just tested locally to be sure

Reply:
That makes an assumption about the their `pwd`, but sure, I'll make the change.